official/recommendation/ncf_keras_main.py

```diff
@@ -121,25 +134,24 @@ def _get_keras_model(params):
   batch_size = params['batch_size']
 
   user_input = tf.keras.layers.Input(
-      shape=(),
-      batch_size=batch_size,
-      name=movielens.USER_COLUMN,
-      dtype=rconst.USER_DTYPE)
-
+      shape=(batch_size,), batch_size=1, name=movielens.USER_COLUMN, dtype=tf.int32)
```

Comment:
is the batch_size argument needed?

Reply:
I believe so. With this argument, the input is of shape (1, 80000), which matches the data.